application/blueprints/datamanager/controllers/transform.py

```
@@ -304,15 +367,285 @@ def handle_check_transform(
+            )
+
+    return features, points
+
+
+def _fetch_boundary_geojson(organisation_code: str) -> dict:
+    empty = {"type": "FeatureCollection", "features": []}
+    try:
+        if ":" not in organisation_code:
+            return empty
+        lpa_prefix, lpa_id = organisation_code.split(":", 1)
+        resp = requests.get(get_entity_search_url(lpa_prefix, lpa_id))
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)